### PR TITLE
1.21 auxpow test set base version

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: test/functional/test_framework/messages.py [hunk 1]
```diff
@@ -632,6 +633,9 @@ def set_null(self):
         self.hash = None
         self.scrypt256 = None
 
+    def set_base_version(self, n):
+        self.nVersion = n
```
Comment (patricklodder) [inline test/functional/test_framework/messages.py:637]: this sets the entire `nVersion` to `n`, not just the base? I assume the "base" is `nVersion & 0x000000ff`?

**Review by patricklodder — COMMENTED** [review]
Am not sure about the actual function doing what it says...

File: test/functional/test_framework/messages.py (hunk 1 above)
Comment (rnicoll) [reply test/functional/test_framework/messages.py:637]: Yeah, there's a follow-on PR that introduces chain IDs, which is the step after this.

**Review by patricklodder — APPROVED** [review]
utACK

**Pull request merged**